packages/zowe-explorer-api/__tests__/__unit__/profiles/ProfilesCache.unit.test.ts

```diff
@@ -309,7 +309,7 @@ describe("ProfilesCache", () => {
             expect((profCache as any).profilesByType.size).toBe(0);
             expect((profCache as any).defaultProfileByType.size).toBe(0);
             expect((profCache as any).allProfiles.length).toBe(0);
-            expect((profCache as any).allTypes).toEqual(["base"]);
+            expect((profCache as any).allTypes).toEqual(["ssh", "base"]);
```

Comment:
Would extenders trying to load `tso` or `zosmf` profiles have the same prroblem?